### PR TITLE
Add an option to set the icon of the window

### PR DESCRIPTION
**Pull request by ulukyn**
Added set_icon method for Qt and CFE Windows

Don't works on Macos

### CODE REVIEW

**Comment by r0x0r** [comment]
@ulukyn Would you address comments?

**Comment by ulukyn** [comment]
Sorry, i'm very busy at the moment, let me answer :)

File: webview/platforms/cef.py [hunk 1]
```diff
@@ -252,6 +253,20 @@ def _create():
 
         instances[window.uid] = browser
         window.shown.set()
+        
+        if window.icon:
+            icon = window.icon+".ico"
```
[the remark this answers is not in the document]
Comment (ulukyn) [reply webview/platforms/cef.py:259]: Windows require .ico, ways to convert a .png to .ico are a little too complex for me. So I think easier to provide both .ico and .png and so, use a sufixless name of icon

File: webview/platforms/cef.py [hunk 2]
```diff
@@ -252,6 +253,20 @@ def _create():
...
+            if os.path.isfile(icon):
+                smallx = windll.user32.GetSystemMetrics(49) #SM_CXSMICON
+                smally = windll.user32.GetSystemMetrics(50) #SM_CYSMICON
+                small_icon =  windll.user32.LoadImageW(0, icon, 1, smallx, smally, 0x00000010)
+                windll.user32.SendMessageW(handle, 0x0080, 0, small_icon)
+
+                bigx = windll.user32.GetSystemMetrics(11) #SM_CXICON
+                bigy = windll.user32.GetSystemMetrics(12) #SM_CYICON
+                big_icon = windll.user32.LoadImageW(0, icon, 1, bigx, bigy, 0x00000010)
+                windll.user32.SendMessageW(handle, 0x0080, 1, big_icon)
```
[the remark this answers is not in the document]
Comment (ulukyn) [reply webview/platforms/cef.py:269]: Ok, sure. I will try to do it this week-end

File: webview/window.py [hunk 3]
```diff
@@ -160,6 +161,10 @@ def get_elements(self, selector):
 
         return self.evaluate_js(code)
 
+    @_shown_call
+    def get_instance(self):
```
[the remark this answers is not in the document]
Comment (ulukyn) [reply webview/window.py:173]: Seems debug code i have used for test, must be removed

File: webview/platforms/qt.py [hunk 4]
```diff
@@ -701,6 +716,9 @@ def get_position(uid):
 def get_size(uid):
     window = BrowserView.instances[uid]
     return window.width(), window.height()
+    
+def get_instance(uid):
```
[the remark this answers is not in the document]
Comment (ulukyn) [reply webview/platforms/qt.py:732]: Seems debug

**Comment by Jzhenli** [comment]
if it's ok to set the icon of the window now?

**Pull request closed without merging**